src/utils.js

```diff
@@ -41,6 +41,20 @@ const isToday = date => {
   return false
 }
 
+const getCurrentShift = date => {
+  let timeOfDay
+
+  const afternoonSplit = 12
+  const eveningSplit = 17
+  const currentHour = moment(date).format('H')
+
+  if (currentHour < afternoonSplit) timeOfDay = 'AM'
+  if (currentHour >= afternoonSplit && currentHour < eveningSplit) timeOfDay = 'PM'
+  if (currentHour >= eveningSplit) timeOfDay = 'ED'
```

Comment:
why do you need this `if` line?  Won't it always be true?

Reply:
i.e. ```
  if (currentHour < afternoonSplit) return 'AM'
  if (currentHour >= afternoonSplit && currentHour < eveningSplit) return 'PM'
  return 'ED'
```

Reply:
also has benefit of getting rid of the mutable variable :nauseated_face:

Reply:
actually even simpler
```
  if (currentHour < afternoonSplit) return 'AM'
  if (currentHour < eveningSplit) return 'PM'
  return 'ED'
```

Reply:
I see what you mean.  Yeah i'm happy with that.